starter_code/queries.md

```diff
@@ -64,20 +73,35 @@
 
 ### 15. All the companies that have been founded on the first seven days of the month, including the seventh. Sort them by their `acquisition price` in a descending order. Limit the search to 10 documents.
 
-<!-- Your Code Goes Here -->
+query: { founded_day : {$lte : 7}}
+sort: {"acquisition.price_amount" : -1}
+skip: 0
+limit: 10
 
 ### 16. All the companies on the 'web' `category` that have more than 4000 employees. Sort them by the amount of employees in ascending order.
 
-<!-- Your Code Goes Here -->
+query: { founded_day : {$lte : 7}}
+sort: {"acquisition.price_amount" : -1}
+skip: 0
+limit: 10
 
 ### 17. All the companies whose acquisition amount is more than 10.000.000, and currency is 'EUR'.
 
-<!-- Your Code Goes Here -->
+query: { founded_day : {$lte : 7}}
+sort: {"acquisition.price_amount" : -1}
+skip: 0
+limit: 10
 
 ### 18. All the companies that have been acquired on the first trimester of the year. Limit the search to 10 companies, and retrieve only their `name` and `acquisition` fields.
 
-<!-- Your Code Goes Here -->
+query: { founded_day : {$lte : 7}}
+sort: {"acquisition.price_amount" : -1}
+skip: 0
+limit: 10
 
 ### 19. All the companies that have been founded between 2000 and 2010, but have not been acquired before 2011.
 
-<!-- Your Code Goes Here -->
+query: { founded_day : {$lte : 7}}
+sort: {"acquisition.price_amount" : -1}
+skip: 0
+limit: 10
```

Comment:
I don't know if it's because you didn't have time, or you just didn't know how to handle these questions, the solution will be pushed during the week, come see us if you have any questions on the last iterations.